meta-resin-common/recipes-core/images/resin-image.bb

```diff
@@ -28,7 +28,6 @@ IMAGE_FEATURES_append = " \
     ${@bb.utils.contains('DISTRO_FEATURES', 'development-image', 'debug-tweaks', '', d)} \
     splash \
     ssh-server-dropbear \
-    read-only-rootfs \
```

Comment:
I recall having issues with resole.conf and needing Write Access to /etc/resolve.conf

Will try flashing with read-only-rootfs set and see if i run into anything